modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc

```diff
@@ -27,12 +31,12 @@ GET /pools/default/certificate
 The `POST` method and `/controller/uploadClusterCA` URI (which are deprecated in 7.1) upload an appropriately configured certificate to the cluster, so that it becomes the root certificate for the cluster.
 Either the Full Admin or the Security Admin role is required.
 The certificate must be specified as binary data, in a location on the filesystem, for successful upload to occur.
-For security reasons, in versions 7.1 and after, by default, this method and URI can be used on _localhost_ only.
+For security reasons, in versions 7.1 and after, by default, this method and URI can be used on _localhost_ only (see xref:rest-api:upload-retrieve-root-cert.adoc#responses[Responses], below, for the associated error message).
 However, this default setting can be changed, by means of the `POST` method and `/settings/security/allowNonLocalCACertUpload` URI, with the `-d false` flag: this also can be used on _localhost_ only.
 The status of the current setting can be retrieved with the `GET` method and `/settings/security/allowNonLocalCACertUpload` URI.
 
 The `GET` method and `/pools/default/certificate` URI are deprecated in 7.1.
-After the cluster has been upgraded to 7.1, they continue to return the certificate that was used prior to upgrade, until at least one new root certificate is uploaded: following which they always return `400 Bad Request`, with the message `deprecated, please use /pools/default/trustedCAs`.
+After the cluster has been upgraded to 7.1, they continue to return the certificate that was used prior to upgrade, until at least one new root certificate is uploaded using the 7.1 API `POST /node/controller/loadTrustedCAs` (as described in xref:rest-api:load-trusted-cas.adoc[Load Root Certificates]): following which they always return `400 Bad Request`, with the message `this API is disabled, please use GET /pools/default/trustedCAs, see documentation for details`.
```

Comment:
After rereading I realized that one more small correction is needed.

> they continue to return the certificate that was used prior to upgrade

It actually now returns the most recent certificate that was uploaded by `POST /controller/uploadClusterCA`.
So incorrectness of current version is in the fact that we can use `POST /controller/uploadClusterCA` **after** the upgrade, and `GET /pools/default/certificate` will return this uploaded cert in this case.

Reply:
I have rewritten this accordingly.